package.json

```diff
@@ -23,7 +23,7 @@
     "test-node": "gulp test",
     "test-browser": "karma start",
     "build": "gulp build",
-    "prepublish": "(not-in-publish && echo 'Skipping prepublish') || npm run require-npm4-to-publish",
+    "prepublish": "require-npm4-to-publish",
     "require-npm4-to-publish": "semver -r '>=4.0.0' $(npm --version) || (echo 'NPM 4+ required to publish, run: sudo npm install -g npm' && exit 1)",
```

Comment:
Does this line need to be removed as well, now that the logic has been moved to an external module?